java/client/build.gradle

```diff
@@ -134,26 +124,82 @@ tasks.register('testFfi', Test) {
 }
 
 tasks.register('buildAll') {
-    dependsOn 'protobuf', 'buildRust', 'testFfi'
+    dependsOn 'protobuf', 'buildRustRelease', 'testFfi'
     finalizedBy 'build'
 }
 
 compileJava.dependsOn('protobuf')
 clean.dependsOn('cleanProtobuf', 'cleanRust')
-test.dependsOn('buildRust')
-testFfi.dependsOn('buildRust')
+
+tasks.register('copyNativeLib', Copy) {
+    from "${projectDir}/../target/release"
+    include "*.dylib", "*.so"
+    into sourceSets.main.output.resourcesDir
+}
+
+jar.dependsOn('copyNativeLib')
+copyNativeLib.dependsOn('buildRustRelease')
+compileTestJava.dependsOn('publishToMavenLocal')
 
 test {
     exclude "glide/ffi/FfiTest.class"
 }
 
+sourceSets {
+    main {
+        java {
+            srcDir 'src/main/java'
+        }
+        resources {
+            srcDir 'src/main/resources'
+        }
+    }
+}
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
+            from components.java
+            groupId = 'software.amazon.glide'
+            artifactId = 'glide'
+            version = "0.0.1"
```

Comment:
change to 255.255.255, we don't want to hardcode the version number of each of the wrappers, instead we change in the CD workflow.
When you'll add a github action to actually publish to maven, we will need to override it inside the workflow